store.go

```diff
@@ -3417,12 +3417,44 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
 	}
 }
 
+var (
+	prevStoreOptions *StoreOptions
+	prevMod          time.Time
+	prevMutex        sync.Mutex
+	prevConfigFile   string
+)
+
+func maybeReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
```

Comment:
nit "reloadConfigurationFileIfNeeded" perhaps?

Reply:
I've used the same naming we have for other functions, but I am fine with changing it